councilmatic_core/management/commands/data_integrity.py

```diff
@@ -1,47 +0,0 @@
-import pysolr
```

Comment:
No longer need this, as we've set up Haystack to remove bills that are no longer in the database.